Language/Dockerfile.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Highlighter
+ *
+ * Copyright (C) 2016, Some right reserved.
+ *
+ * @author Kacper "Kadet" Donat <kacper@kadet.net>
+ *
+ * Contact with author:
+ * Xmpp: me@kadet.net
+ * E-mail: contact@kadet.net
+ *
+ * From Kadet with love.
+ */
+
+namespace Kadet\Highlighter\Language;
+
+use Kadet\Highlighter\Matcher\RegexMatcher;
+use Kadet\Highlighter\Parser\Rule;
+
+class Dockerfile extends Shell
+{
+    public function setupRules()
+    {
+        parent::setupRules();
+
+        $keywords = [
+            'ADD', 'ARG', 'AS', 'CMD', 'COPY', 'ENTRYPOINT', 'ENV', 'EXPOSE', 'FROM',
+            'HEALTHCHECK', 'LABEL', 'MAINTAINER', 'ONBUILD', 'RUN', 'SHELL', 'STOPSIGNAL',
+            'USER', 'VOLUME', 'WORKDIR',
+        ];
+
+        $this->rules->add('keyword', new Rule(
+            new RegexMatcher('/^\s*\b(' . implode('|', $keywords) . ')\b/mi'),
```

Comment:
Not sure if that `\b` is required since it must be first word of a line? The same goes for the last one - it should probably be `\s` as I don't think that syntax like `RUN["test"]` is a valid one? Not sure however.